carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -369,6 +374,115 @@ void WMBroadcaster::removeGeofence(std::shared_ptr<Geofence> gf_ptr)
   carma_wm::toBinMsg(send_data, &gf_msg_revert);
   map_update_pub_(gf_msg_revert);
 };
+  
+void  WMBroadcaster::routeCallbackMessage(const cav_msgs::Route& route_msg)
+{
+
+ cav_msgs::ControlRequest cR; 
+ cR =  routeCallbackMessageLogic(route_msg);
+ route_callmsg_pub_(cR);
+
+}
+
+
+cav_msgs::ControlRequest WMBroadcaster::routeCallbackMessageLogic(const cav_msgs::Route& route_msg) 
+{
+
+
+  auto path = lanelet::ConstLanelets(); 
```

Comment:
You could just declare this object using the data type shown. No need for an assignment with the constructor. 
```
lanelet::ConstLanelets path;
```